splash/browser_tab.py

```diff
@@ -299,6 +301,16 @@ def go(self, url, callback, errback, baseurl=None, http_method='GET',
         if body is not None:
             body = to_bytes(body)
 
+        if headers is not None:
+            # headers can be list or dict, keys can be bytes or str, HTTP specs
+            # say headers are case-insensitive, so they can be uppercase or lowercase
+            if not isinstance(headers, dict):
+                headers = dict(headers)
+            headers = {to_bytes(k.lower()): to_bytes(v) for k, v in headers.items()}
```

Comment:
Wouldn't this make all headers lower-case? While HTTP specs says headers are case-insensitive we shouldn't change case of headers if user passes them.